reporters/GlobalSummaryReporter.js

```diff
@@ -130,11 +131,6 @@ class GlobalSummaryReporter {
     console.group();
     console.log(bold("Token summary stats"));
     console.log(italic("- Token price is sourced from exchange where synthetic token is traded (i.e. Uniswap)"));
-    console.log(
```

Comment:
This message is intended for devs and is already described by a comment in `./index.js`